app/models/katello/glue/pulp/consumer.rb

```diff
@@ -197,7 +197,7 @@ def katello_agent_installed?
       def fetch_package_profile
         Katello.pulp_server.extensions.consumer.retrieve_profile(uuid, 'rpm')
       rescue RestClient::ResourceNotFound => e
-        Rails.logger.info "Failed to find profile for #{uuid}: #{e}}"
+        Rails.logger.error "Failed to find profile for #{uuid}: #{e}, #{e.backtrace.join("\n")}"
```

Comment:
So I altered this last week in an effort to suppress a backtrace, but what the bug really wanted was to suppress the error altogether. Since I'm changing the way this method is called, I thought I should change it back to its original format. I can keep it on info if you like, but it was only changed for this specific bug.